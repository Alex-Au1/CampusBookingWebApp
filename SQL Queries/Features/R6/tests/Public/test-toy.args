{
    "args": [],
    "kwargs": {
        "roomName": "MC 1085",
        "minCapacity": "100",
        "maxCapacity": "200",
        "startTimeStr": "2025-06-23 17:46:05",
        "endTimeStr": "2025-06-23 20:07:05"
    }
}   