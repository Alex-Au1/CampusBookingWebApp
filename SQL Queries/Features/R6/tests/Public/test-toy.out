{'roomID': UUID('00000000-0000-0000-0000-000000000006'), 'buildingName': 'MC', 'roomName': 'MC 1085', 'overlappingBookings': 0, 'capacity': 200, 'addressLine1': '200 University Avenue West', 'addressLine2': 'MC', 'city': 'Waterloo', 'province': 'Ontario', 'country': 'Canada', 'postalCode': 'N2L 3G1'}